pennylane/ops/qubit/__init__.py

```diff
@@ -0,0 +1,106 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+This module contains the discrete-variable quantum operations.
+
+The operations are divided into six different files:
+
+* ``arithmetic_ops.py``: Operations that perform arithmetic on the states
```

Comment:
Maybe every description could be a sentence?
```suggestion
* ``arithmetic_ops.py``: Operations that perform arithmetic on the states.
```

Reply:
Played around with making the descriptions into sentences, but then it was just a bunch of "this file contains ... " unnecessarily wordy sentences. Am tidying the descriptions up a little though.